media_commons_booking_app/webpack.config.js

```diff
@@ -301,7 +301,7 @@ const clientConfigs = clientEntrypoints.map((clientEntrypoint) => {
       }),
       new HtmlWebpackPlugin({
         template: clientEntrypoint.template,
-        filename: `${clientEntrypoint.filename}${isProd ? '' : '-impl'}.html`,
+        filename: `${clientEntrypoint.filename}${isProd ? '' : ''}.html`,
```

Comment:
@rlho can answer better than I. something (I can't recall the error) was prohibiting Riho and I from running the app locally and that seemed to fix it. shall we put it back and see if we can replicate?

Reply:
@lucia-gomez the error in question was in the browser. I'm getting it now when I try to run your `refactorUi` branch locally.

`The webpage at https://localhost:3000/index-impl.html might be temporarily down or it may have moved permanently to a new web address.`

![image](https://github.com/ITPNYU/mc-reservation-tool/assets/688639/dc68b56b-c61a-4727-973f-a08f92169a07)